definitions/npm/rethinkdb_v2.x.x/flow_v68.x-/test_rethinkdb_v2.x.x.js

```diff
@@ -0,0 +1 @@
+// TODO write some tests!
```

Comment:
so... do it. ;-) Write at least a few as a smoke screen, and remember to use describe/it blocks.